static/open-specs/chat.yaml

```diff
@@ -1073,3 +1077,98 @@ paths:
                   code: 50001
                   statusCode: 500
                   href: "https://help.ably.io/error/50001"
+  /chat/{version}/rooms/{roomName}/messages/{serial}/client-reactions:
+    get:
+      summary: Get reactions summary for a specific client
+      description: |
+        Gets the reactions summary for a specific client ID.
+
+        A successful request returns a clipped message reactions summary for the specific client ID. The list of client IDs on all reaction types will be clipped to only include the requested user.
+      tags:
+        - rooms
+      parameters:
+        - name: version
+          in: path
+          required: true
+          description: The version of the Chat API endpoint to use.
+          schema:
+            type: string
+            example: v3
+        - name: roomName
+          in: path
+          required: true
+          description: |
+            The unique identifier of the room containing the message with the reaction you wish to delete.
+            The `roomName` must be URI-encoded.
+          schema:
+            type: string
+        - name: serial
+          in: path
+          required: true
+          description: |
+            The unique identifier of the message containing the reaction you wish to delete.
+            The `serial` must be URI-encoded.
+          schema:
+            type: string
+            example: "01826232498871-001@abcdefghij:001"
+        - name: forClientId
+          in: query
+          required: false
+          description: |
+            If requesting data for a client ID other than the authenticated one, provide the required client ID in this parameter.
+          schema:
+            type: string
+            example: "like"
+        - $ref: '#/components/parameters/ClientIdHeader'
+        - $ref: '#/components/parameters/AuthorizationHeader'
+        - $ref: '#/components/parameters/AcceptHeader'
```

Comment:
See David's PR here: https://github.com/ably/docs/pull/2840/files

We should also add ClientIdParam